test/test_gesvd.cc

```diff
@@ -38,10 +38,10 @@ void test_gesvd_work( Params& params, bool run )
     //params.gflops();
     params.ortho_U();
     params.ortho_V();
-    params.error_sigma();
+    params.error2();
+    params.error2.name( "Sigma" );
     params.msg();
 
-    params.error.name( "A - USV^H\nerror" );
```

Comment:
Stupid question: why is the name erased?

Reply:
Most routines have "error" as a generic column header. We didn't usually try to write the formulas like b-Ax or PA-LU or A-LL^H, partly because the formula is more complicated than that with normalization, etc.

Actually, `geev` and `gehrd` that you also commented on are the only routines that set the name of the error field to something else. For `geev` that makes a lot of sense because there's an error for left eigvec and an error for right eigvec. `gehrd` could just use the usual "error" and "orth." labels instead of "A-UHU^H" and "I-UU^H".

Reply:
I see. Thank you for the explanation.